articles/api-auth/dynamic-client-registration.md

```diff
@@ -0,0 +1,205 @@
+---
+description: How to dynamically register Clients with Auth0, using the Management APIv2.
+toc: true
+---
+
+# Dynamic Client Registration
+
+Dynamic Client Registration is a feature, based on the [OpenID Connect Dynamic Client Registration specification](https://openid.net/specs/openid-connect-registration-1_0.html), that enables you to to register clients dynamically, using the [Management APIv2](/api/management/v2). This way you can allow third party developers to register and use clients.
+
+All clients, registered dynamically with Auth0, have the following characteristics:
+
+- The [ID tokens](/tokens/id-token) generated for these clients, hold minimum user profile information. In order to retrieve the full user profile, you need to get an access token and use the [/userinfo endpoint](/api/authentication#get-user-info).
+
+- These clients cannot use [ID tokens](/tokens/id-token) to invoke [Management APIv2](/api/management/v2) endpoints. Instead, they should [get a Management APIv2 Token](/api/management/v2/tokens#how-to-get-a-management-apiv2-token). Note that the client should be granted the `current_user_*` scopes, as required by each endpoint.
+  - `read:current_user`: [List or search users](/api/management/v2#!/Users/get_users), [Get a user](/api/management/v2#!/Users/get_users_by_id), [Get user Guardian enrollments](/api/management/v2#!/Users/get_enrollments)
+  - `update:current_user_metadata`: [Update a user](/api/management/v2#!/Users/patch_users_by_id), [Delete a user's multifactor provider](/api/management/v2#!/Users/delete_multifactor_by_provider)
+  - `delete:current_user_metadata`: [Delete a user](/api/management/v2#!/Users/delete_users_by_id)
+  - `create:current_user_metadata`: [Create a user](/api/management/v2#!/Users/post_users)
+  - `create:current_user_device_credentials`: [Create a device public key](/api/management/v2#!/Device_Credentials/post_device_credentials)
+  - `delete:current_user_device_credentials`: [Delete a device credential](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id)
+  - `update:current_user_identities`: [Link a user account](/api/management/v2#!/Users/post_identities), [Unlink a user identity](/api/management/v2#!/Users/delete_provider_by_user_id)
+
+- They cannot have any connections, except for tenant level connections (domain connections). To use tenant level connections, you need the latest version of [Lock](/libraries/lock).
```

Comment:
Rephrase. `If the client wants to use Lock to authenticate users, it will need to use a version greater than 10.7`

Note, clients don't _have_ connections. They can consume a connection, but it's not like an intrinsic property of the client. The picture gets clearer with "Domain Connections". These are consumed by third party clients but they have no other relationship whatsoever.